lxd/containers.go

```diff
@@ -35,8 +36,7 @@ func containersPost(d *Daemon, r *http.Request) Response {
 
 	name, err := raw.GetString("name")
 	if err != nil {
-		/* TODO: namegen code here */
-		name = "foo"
+		name = petname.Generate(2, "-")
```

Comment:
Looking at the golang-petname generator, some names may include caps, it'd probably make sense to ensure this is all lowercase.